assets/_scss/elements/_typography.scss

```diff
@@ -116,7 +116,7 @@ dfn {
 }
 
 .usa-sans,
-.usa-sans p {
+.usa-sans * {
```

Comment:
We could change this so its all text elements, `<a>`, `<p>`, `<h1>`, etc. Or we could change the code so you add the class to the actual element you want to modify rather then the parent (which would be more verbose but safer.